### PR TITLE
CYOA Homework

### PR DESCRIPTION
**Pull request by Cajam**
Completeness: 4/5 

Comfort: 4/5

I figured out how to do branching paths pretty quickly. I had more of a problem thinking of a concept that would require a loop, which is why I don't think my loop is very good, even though I feel as though I understand how to write one decently.

### CODE REVIEW

File: starter/script.js [hunk 1]
```diff
@@ -1 +1,42 @@
-// put code here!
+var genrechoice = prompt("You are settling in for a night of Netflix and chill. What kind of movie do you want to watch: romance, action, or something else?");
```
Comment (jshawl) [inline starter/script.js:1]: Such a great app idea. People would pay money for this i bet.

File: starter/script.js [hunk 2]
```diff
@@ -1 +1,42 @@
...
+  if (genrechoice == "romance") {
+    romance = prompt("Hot stuff. Do you want it to be funny or dramatic?")
+      if (romance == "funny") {
+        funny = prompt("Ok, do you like Mila Kunis or Amy Schumer?")
+          if (funny == "Mila Kunis"){
+            document.write("Mila Kunis is my favorite, so we're going to watch Forgetting Sarah Marshall. Have fun!")
+          } else if (funny == "Amy Schumer") {
```
Comment (jshawl) [inline starter/script.js:8]: ha!

**Comment by jshawl** [comment]
:+1: :star: :star2: this is a clever and funny concept cassidy!! Great job.

**Comment by Cajam** [comment]
Thanks! I was inspired by a website that helps you pick Netflix movies.
Maybe I have a future in app development to put that on mobile, haha.

Very respectfully,

_Cassidy J. Harpster_
cassidy.j.harpster@gmail.com
(850) 218-4673
Washington, D.C.

**Pull request closed without merging**